tests/benchmarks/test_benchmark_ddpg.py

```diff
@@ -2,8 +2,8 @@
 This script creates a regression test over garage-DDPG and baselines-DDPG.
```

Comment:
just a thought: perhaps we should add a test case to each benchmark which runs each benchmark for just 1 iteration, to make sure that all the benchmarks stay running using the CI.

we would still exclude the main benchmark from CI

Reply:
Good idea. I believe Kevin has done a similar thing in his branch.

Reply:
@cheng-kevin I assume the files under [this folder](https://github.com/rlworkgroup/garage/tree/Automated_Benchmarking_normalized/tests/benchmarks/continuous_integration) are for ci? Could you port them to this branch?